theano/sandbox/solve.py

```diff
@@ -24,7 +24,8 @@ class Solve(gof.Op):
     #     sym_pos, lower, overwrite_a, overwrite_b
 
     # TODO: Add C code that calls the underlying LAPACK routines
-    #      and keeps a memory workspace from call to call as a non-default Op output
+    #      and keeps a memory workspace from call to call
```

Comment:
This line seems too short. Make sure you break the line at the last position possible